assets/src/pages/room/consts.ts

```diff
@@ -28,6 +28,8 @@ const VIDEO_TRACK_CONSTRAINTS: MediaStreamConstraints = {
 const SCREENSHARING_MEDIA_CONSTRAINTS: MediaStreamConstraints = {
   video: {
     frameRate: { ideal: 10, max: 15 },
+    width: { max: 1920, ideal: 1920 },
+    height: { max: 1280, ideal: 1280 },
```

Comment:
I've seen that before and comes from Daniel's PR, I guess a rebase/merge of master is needed